ctx_test.go

```diff
@@ -5474,11 +5474,11 @@ func Test_GenericParseTypeInt64s(t *testing.T) {
 		},
 		{
 			value: int64(math.MaxInt64),
-			str:   strconv.Itoa(math.MaxInt64),
+			str:   strconv.FormatInt(math.MaxInt64, 10),
```

Comment:
This is not needed, golang already does this inside Itoa

https://go.dev/src/strconv/itoa.go